### ISSUE TITLE
Empty Directories

### ISSUE THREAD

**Issue by pyeager56** [bluecherrydvr/bluecherry-apps#578]
When files are deleted by bc-server, the directories are left behind.

It would seem that at some point this might lead to a shortage of inodes.

**Comment by curtishall** [comment]
Yes, we should delete the directories once they are empty.  I'm not overly worried about the inodes...you would have to have an amazing amount of cameras over large periods of time to come close to reaching any inode issues.

**Comment by kala13x** [comment]
@curtishall  just pushed changes to the separate branch: https://github.com/bluecherrydvr/bluecherry-apps/tree/cleanup_dir
I will double test it and then open pull request.
